### ISSUE TITLE
Notebook_Inference_clustering

### ISSUE THREAD

**Issue by anzonyqr** [alexanderquispe/14.38_Causal_ML#47]
We have to replicate this [kaggle notebook](Notebook_Inference_clustering).

**Comment by anzonyqr** [comment]
### cv.glmnet curiosities
1. According to [link1](https://stackoverflow.com/questions/24098212/what-does-the-option-normalize-true-in-lasso-sklearn-do) and [link2](https://statisticaloddsandends.wordpress.com/2018/11/15/a-deep-dive-into-glmnet-standardize/), It standardize **X** variables before estimation. In sklearn we have an option named **normalize**. It normalizes **X** by subtracting the mean and dividing by the l2-norm. Based on [link3](https://stackoverflow.com/questions/59846325/confusion-about-standardize-option-of-glmnet-package-in-r), **cv.glmnet (standardize = TRUE)** and **sklearn (normlize = True)** are not the same. 

### Random Forest curiosities
We added some extra hyperparameters to RF models to be equal to the R default hyperparameters.
